packages/admin-client/src/common/components/PdfButtonField/PdfButtonField.tsx

```
@@ -19,10 +19,49 @@ const PdfButtonField: React.FC<PDFButtonFieldProps> = ({ record, resource }) =>
                 id: record?.id,
                 formType: formType
             })
-            .then(({ result }) => {
-                const filename =
-                    (resource === "applications" ? "application_" : "claim_") + record?.form_confirmation_id + ".pdf"
-                downloadPdf(result, filename)
+            .then(async ({ result }) => {
+                if (resource === "claims") {
+                    try {
+                        const { employerInfo } = await dataProvider.getEmployerInfo("employer", { id: record?.id })
+                        if (employerInfo) {
```

if `employerInfo` is null then nothing in the `if` block will get executed and no error will be reported (the user will just be confused as to why nothing is happening). Instead you should throw an error if `employerInfo` is null, and if not then execute the logic. This avoids putting all the logic in an `if` block as well (since execution will stop once you throw an error and go to the `catch` portion of the `try/catch`).